static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1144,6 +1145,28 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     );
   }
 
+  renderFeedbackBanner() {
```

Comment:
can we make this it's own (functional) component?

Reply:
<img src="https://media3.giphy.com/media/Ge0Bt025YVlhXzOmuP/giphy.gif"/>